src/IO/TinyPlyLoader/TinyPlyFileLoader.cpp

```diff
@@ -74,6 +75,66 @@ struct memory_stream : virtual memory_buffer, public std::istream {
         memory_buffer( first_elem, size_ ), std::istream( static_cast<std::streambuf*>( this ) ) {}
 };
 
+template <typename DataType, typename ContainerType>
+void copyArrayToRadiumcontainer( const uint8_t* buffer, ContainerType& container, size_t count ) {
```

Comment:
```suggestion
void copyArrayToContainer( const uint8_t* buffer, ContainerType& container, size_t count ) {
```
I don't get why the container is "Radium" here